kfs-ld/src/main/java/edu/arizona/kfs/module/ld/LaborConstants.java

```diff
@@ -0,0 +1,30 @@
+package edu.arizona.kfs.module.ld;
+
+public class LaborConstants extends org.kuali.kfs.module.ld.LaborConstants {
+
+	// EreSweep
+	public static final String FRINGE_FINANCIAL_BALANCE_TYPE_CODE = "CB";
+	public static final String FRINGE_FINANCIAL_OBJECT_TYPE_CODE = "EX";
+	public static final String FRINGE_FINANCIAL_DOCUMENT_TYPE_CODE = "FRNS";
+	public static final String FRINGE_TRANSACTION_LEDGER_ENTRY_DESCRIPTION = "Fringe Budget Sweep";
+	public static final String FRINGE_FINANCIAL_OBJECT_CODE = "2000";
+	public static final String FRINGE_BALANCE_TYPE_CODE_CB = "CB";
+	public static final String FRINGE_BALANCE_TYPE_CODE_AC = "AC";
+	public static final String FRINGE_BALANCE_TYPE_CODE_IE = "IE";
+	public static final String EMPTY_SUB_ACCOUNT_STRING = "-----";
+	public static final String EMPTY_SUB_OBJECT_STRING = "---";
+	public static final String BALANCE_TYPE_CURRENT_BUDGET = "CB";
+	public static final String BALANCE_TYPE_CURRENT_BUDGET_LABEL_DESCRIPTION = "Current Budget";
+	public static final String BALANCE_TYPES = "BALANCE_TYPES";
+	public static final String BALANCE_TYPES_NAMESPACE = "KFS-CAB";
+	public static final String BALANCE_TYPES_COMPONENT = "Batch";
+	public static final String PENDING_ENTRY_DEFAULT_ACCOUNT_NUMBER = "XXXX";
+	public static final String DOCUMENT_NUMBER_PREFIX = "SWADJ-";
+	public static final String ERE_SWEEP_DATA = "EreSweep.data";
+	public static final String ERE_SWEEP_ERROR_DATA = "EreSweepError.data";
+	public static final String ERE_SWEEP_DONE = "EreSweep.done";
+	public static final String ERE_SWEEP_RECON = "EreSweep.recon";
+	public static final String END_OF_LINE = ";\n";
```

Comment:
The '\n' is a UNIX newline, and I wonder what would happen here if we ran on say, Azure. Consider perhaps doing what line 28 does, which could be something like:

`String.format(";%n")`

Which should then should behave as expected on all platforms.